pliers/extractors/models.py

```diff
@@ -84,3 +84,65 @@ def _extract(self, stim):
             values.append(float(extraction[1]))
 
         return ExtractorResult([values], stim, self, features=features)
+
+
+def _resize_image(image, shape):
```

Comment:
This can probably wait, but I imagine that changes in aspect ratio will tend to throw the model off, so ideally the resizing here should maintain the aspect ratio of the input and pad with empty borders as needed. I don't remember how or if this was handled in the previous extractor (I certainly didn't do it myself, but perhaps it's done internally).

Reply:
On further thought, in principle, I think this is exactly the kind of problem that we have the `Filter` hierarchy for... it makes sense to me to implement an `ImageResizingFilter` that basically wraps this function and optionally adds padding etc. Again, not so much a suggestion for present action as a note to self to do this in future (and may be worth opening an issue for).

Reply:
I haven't tested this, but I don't think changing the aspect ratio should affect the prediction too much. The model is definitely trained with augmentation, which includes distorting the aspect ratios of the images. In the tensorflow/keras [tutorial](https://www.tensorflow.org/tutorials/images/hub_with_keras#run_it_on_a_single_image) on image classification, they use the Pillow resize method, which doesn't maintain aspect ratio. That being said, I definitely can add an `ImageResizingFilter` that can resize while optionally maintaining aspect ratio. I have the code written, but I'll submit a different PR for it.

Reply:
#342 adds the `ImageResizingFilter`